packages/contracts/contracts/ArtonomousCurvedBond.sol

```diff
@@ -1,14 +1,16 @@
 pragma solidity ^0.4.24; //solhint-disable-line compiler-fixed
 
+import './EthBondingCurvedToken.sol';
 
-contract ArtonomousCurvedBond is EthPolynomialCurvedToken {
+contract ArtonomousCurvedBond is EthBondingCurvedToken('Artonomous SOUL', 18, 'SOUL') {
 
     /// @dev constructor        Initializes the bonding curve
     /// @param name             The name of the token
     /// @param decimals         The number of decimals to use
     /// @param symbol           The symbol of the token
     /// @param _exponent        The exponent of the curve
-    constructor() EthBondingCurvedToken('Artonomous SOUL', 18, 'SOUL') public {
+    uint exponent;
+    constructor() public {
```

Comment:
This is incorrect however, since the constructor needs to call its "super". Did dit his also fail to compile?

Reply:
it looks like `EthPolynomialCurvedToken.sol` has an `exponent` variable, but not the `EthBondingCurvedToken.sol`.  should `ArtonomousCurvedBond` extend EPCT, or does it not actually need an exponent?